src/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Apr 26 12:53:22 EDT 2017
+4=\#
+3=org.springframework
+2=
+1=javax
+0=java
```

Comment:
I was able to reproduce by switching to the IDEA formatter (using the switch button) and optimizing imports. It rearranges the imports in a different way. When I change back to the Eclipse formatter, the right order is adopted. I wonder if this is also your case?